tabs/src/components/sample/QueryRunner.jsx

```diff
@@ -67,25 +69,62 @@ export function QueryRunner() {
         items: ['Key', 'Value'],
     };
 
-    const callGraph = () => {
-        // stub
-        console.log(requestType);
-        console.log(graphVersion);
-        console.log(query);
-        setResponseBody("");
-        setResponseHeaders([{ key: 'Content', items: ['Content', 'json'] }]);
-    };
+    async function callGraph() {
+        const queryParameters = query.substring(GRAPH_URL.length + graphVersion.length, query.length);
+        const url = RSC_API_URL + graphVersion + queryParameters;
+        const cleanedHeaders = {};
+        for (const i of requestHeaders) {
+            cleanedHeaders[i.items[0]] = i.items[1];
+        }
```

Comment:
```suggestion
        for (const header of requestHeaders) {
            cleanedHeaders[header.items[0]] = header.items[1];
        }
```

nit: for easier readability

Reply:
ac5e50a6af607d75f2f13ce91cea9eedd89751ea